yml/dp02_dc2.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-name: dp02Schema
-"@id": "#dp02Schema"
+name: dp02_dc2_catalogs
+"@id": dc2
```

Comment:
I think this string should match the schema name, especially if we are also going to have a `dp02_dc2_images` schema added that needs to be distinguished from `dp02_dc2_catalogs`.